mcos/covariance_transformer.py

```diff
@@ -197,3 +186,15 @@ def _corr_to_cov(self, corr: np.array, std: np.array) -> np.array:
         """
         cov = corr * np.outer(std, std)
         return cov
+
+
+def cov_to_corr(cov: np.array) -> np.array:
+    """
+    Derive the correlation matrix from a covariance matrix
+    :param cov: covariance matrix
+    :return: correlation matrix
+    """
+    std = np.sqrt(np.diag(cov))
+    corr = cov / np.outer(std, std)
+    corr[corr < -1], corr[corr > 1] = -1, 1  # numerical error
+    return corr
```

Comment:
Since this is used by 2 different modules , maybe we should extract this to a `utils.py` file? I'm sure we'll come across more helper methods in the future.

Reply:
I think it's fitting in the `covariance_transformer.py` file since it's transforming a covariance matrix, what do you think?

Reply:
Yea that makes sense toe me!